serverside/zerotrace/example/main.go

```diff
@@ -30,6 +33,42 @@ var (
 	l                = log.New(os.Stderr, "example: ", log.Ldate|log.Ltime|log.LUTC|log.Lshortfile)
 )
 
+// getMinRttValue gets the minimum RTT value from the array
+// if nwLayerRttTCP fails, it returns -1, 
```

Comment:
Not sure what these comments are about? All this function does is return the minimum?

Reply:
I've fixed the comments in 75c4cf5

Reply:
I can't find this commit. Did you push it already?

Reply:
Sorry! I had commits lined up that I thought I pushed, but they're pushed now.